src/services/UserMetricsService.ts

```diff
@@ -0,0 +1,51 @@
+import { Repository } from 'typeorm';
+import { MysqlDataSource } from '../config/database';
+import { Metrics } from '../database/entity/UserMetric';
+
+export class UserMetricsService {
+  private metricRepository: Repository<Metrics>;
+
+  constructor() {
+    this.metricRepository = MysqlDataSource.getRepository(Metrics);
+  }
+
+  async findByMetric(
+    metric: 'Registrations Incompleted' | 'Registrations Started'
+  ): Promise<Metrics | undefined> {
+    return await this.metricRepository.findOne({ where: { metric } });
+  }
+
+  async updateRegSta(
+    metric: 'Registrations Started'
+  ): Promise<void | undefined> {
+    const regSta = await this.metricRepository.findOne({ where: { metric } });
+    if (regSta) {
+      regSta.quantity = regSta.quantity + 1;
+      this.metricRepository.save(regSta);
+    }
+  }
+
+  async updateRegInc(
```

Comment:
Não faria mais sentido ter apenas um método que faz o incremento da metrica que for passada? Seria bacana usar o método específico de increment que tem no typeorm: https://typeorm.io/repository-api

Reply:
Feito.